<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GirderFileBrowserDialog</class>
 <widget class="QDialog" name="GirderFileBrowserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Girder File Browser</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="horizontalSpacing">
    <number>9</number>
   </property>
   <item row="2" column="2">
    <widget class="QPushButton" name="push_goHome">
     <property name="text">
      <string>Go Home</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="push_goUpDir">
     <property name="text">
      <string>Go Up</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QListView" name="list_fileBrowser">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="layout_rootPath">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>15</number>
     </property>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QComboBox" name="combo_itemMode">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Treat items as files, or as folders?&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&amp;quot;Treat Items as Files&amp;quot; may be significantly faster over a slow internet because it requires fewer api calls.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&amp;quot;Treat Items as Folders&amp;quot; will allow one to browse files within items.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&amp;quot;Treat Items as Folders&amp;quot; will also check the contents of every item in the current folder. If there is one file in the item and it has the same name as the folder, the file will be moved up one level and the item ignored.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Treat Items as Files</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Treat Items as Folders</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
